### PR TITLE
Hide filtering of teams for unsigned users

### PR DESCRIPTION
**Pull request by lheilmann**
See #356

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -44,6 +44,8 @@
   create_table "events_users", id: false, force: :cascade do |t|
     t.integer "event_id", null: false
     t.integer "user_id", null: false
+    t.index ["event_id", "user_id"], name: "index_events_users_on_event_id_and_user_id"
```
Comment (birneamstiel) [inline db/schema.rb]: I was wondering: Is there an explanation for why this is in here?

**Review by birneamstiel — APPROVED** [review]
LGTM!
(haven't tested yet)

File: db/schema.rb (hunk 1 above)
Comment (lheilmann) [reply db/schema.rb]: No idea! Probably generated after `rails db:migrate`. What would you prefer? Ignore it or keep it out of this PR?

File: spec/views/teams/index.html.erb_spec.rb [hunk 2]
```diff
@@ -39,4 +39,22 @@
     render
     expect(rendered).to have_selector(:link_or_button, t('teams.index.create_team'))
   end
+
+  it "renders the delete button when signed in and you are an admin" do
```
Comment (birneamstiel) [inline spec/views/teams/index.html.erb_spec.rb]: @lheilmann we think this test doesn't belong here... It probably got in accidentally when resolving merge conflicts.

**Review by birneamstiel — CHANGES_REQUESTED** [review]

File: spec/views/teams/index.html.erb_spec.rb (hunk 2 above)
Comment (lheilmann) [reply spec/views/teams/index.html.erb_spec.rb]: @birneamstiel You're right! Should I exlude it from this PR?

**Review by birneamstiel — APPROVED** [review]

**Pull request merged**